go-controller/pkg/ovn/egressip_test.go

```diff
@@ -6710,18 +6862,19 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 					Ports: []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1.Name + "-UUID"},
 					Nat:   []string{"egressip-nat-UUID1"},
 				}
-				nodeIPsASdbIDs := getEgressIPAddrSetDbIDs(NodeIPAddrSetName, DefaultNetworkControllerName)
-				fakeOvn.asf.EventuallyExpectAddressSetWithAddresses(nodeIPsASdbIDs, []string{node1IPv4})
-				egressSvcPodsV4, _ := addressset.GetHashNamesForAS(egresssvc.GetEgressServiceAddrSetDbIDs(DefaultNetworkControllerName))
-				egressipPodsV4, _ := addressset.GetHashNamesForAS(getEgressIPAddrSetDbIDs(EgressIPServedPodsAddrSetName, DefaultNetworkControllerName))
-				nodeIPsV4, _ := addressset.GetHashNamesForAS(nodeIPsASdbIDs)
 				nodeSwitch.QOSRules = []string{"egressip-QoS-UUID"}
+
+				egressSVCServedPodsASv4, _ := buildEgressIPServiceAddressSets(nil)
+				egressIPServedPodsASv4, _ := buildEgressIPServedPodsAddressSets([]string{oldEgressPodIP})
+				egressNodeIPsASv4, _ := buildEgressIPNodeAddressSets([]string{node1IPv4})
+				namespaceAddressSetv4, _ := buildNamespaceAddressSets(eipNamespace, []string{"10.128.0.50"})
```

Comment:
same here with 0.50 is that a hardcoded nodeIP or podIP etc?

Reply:
took another look, this happens because the test author did not specify an address for the pod, this is the next address assigned. Luckily they grabbed the address from the pod so I used that instead of the hardcode